src/compiler/emitter.ts

```diff
@@ -5565,12 +5565,12 @@ var __param = (this && this.__param) || function (paramIndex, decorator) {
                 // we need to add modules without alias names to the end of the dependencies list
 
                 let aliasedModuleNames: string[] = [];   // names of modules with corresponding parameter in the
-                                                         // factory function.
+                // factory function.

```

Comment:
Agree. This kind of comment (required breaking into two line in this manner) should be move above